qemu/tests/cfg/disk_extension.cfg

```diff
@@ -40,6 +40,7 @@
     virtio_blk:
         driver_name = viostor
     virtio_scsi:
+        cd_format_cd1 = ide
```

Comment:
From your commit :
```
There is error "scsi-cd is not supported by data plan" when
set driveformat to virtio_scsi for Windows guest testing.
Change cdrom format to ide to fix this issue.
```
So you wanna let it works only windows, maybe  you should limit it with `Windows:`.
Thanks.

Reply:
Updated.